usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -120,6 +124,26 @@ LEFT JOIN toptier_agency TAA ON (AA.toptier_agency_id = TAA.toptier_agency_id)
 LEFT JOIN subtier_agency SAA ON (AA.subtier_agency_id = SAA.subtier_agency_id)
 LEFT JOIN toptier_agency TFA ON (FA.toptier_agency_id = TFA.toptier_agency_id)
 LEFT JOIN subtier_agency SFA ON (FA.subtier_agency_id = SFA.subtier_agency_id)
+-- The next four joins are needed to get the Agency ID for all combinations of Awarding / Funding
```

Comment:
Not tracking why this is necessary

Reply:
When hitting the `awarding_agency` endpoint the `awarding_agency_id` is the ID from Agency table, but it is not the correct ID for the response since the `toptier_flag` might be False for that particular Agency. Needed to take the `toptier_flag` into account when pulling the ID that matches the Agency record. For example, that are dozens of records with the `toptier_agency_id = 68` in the Agency table, but only one of them has the `toptier_flag = True`.

Reply:
Oh right, that's a new concern. Let me shoot you something outside Github to brainstorm something